cl/users/signals.py

```diff
@@ -33,6 +45,41 @@ def get_message_id(mail_obj: dict) -> str:
     return ""
 
 
+def store_bounce_or_complaint_obj(
+    bounce_obj: dict[str, Any], obj_type: SESEventType
+) -> None:
+    """
+    Store the SES bounce/complaint payload to S3 as JSON.
+
+    :param bounce_obj: The bounce payload to store.
+    :param obj_type: The type of the bounce/complaint.
+    """
+
+    match obj_type:
+        case SESEventType.COMPLAINT:
+            dir_name = "complaints"
+            storing_rate = settings.COMPLAINTS_STORE_RATE
+        case SESEventType.BOUNCE:
+            dir_name = "bounces"
+            storing_rate = settings.BOUNCES_STORE_RATE
+        case _:
+            return None
+
+    if random.random() >= storing_rate:
+        # Ignore events over the random rate.
+        return None
+
+    try:
+        storage = S3PrivateUUIDStorage()
+        feedback_id = bounce_obj.get("feedbackId", "unknown_id")
```

Comment:
Is the feedback_id a useful value to humans? I wonder if we could prefix these by the email address or something, since that's how we'd usually want to look things up? 

Maybe `email_address/feedback_id.json`?

Reply:
Sure! I've applied this suggestion. Bounces and complaints will now be stored at:

`com-courtlistener-private-storage/complaints/email_address@domain.com/unique.id.json`
`com-courtlistener-private-storage/bounces/email_address@domain.com/unique.id.json`

> Is the feedback_id a useful value to humans?

I don't think so. Actually, since I used the private storage class, the file name is being overridden by a unique ID.